src/az390.java

```diff
@@ -10923,6 +11005,196 @@ private void fp_get_hex(){
 				dc_hex = "8" + dc_hex.substring(1);
 			}
 			return;
+
+		} else if (fp_text.toUpperCase().equals("(INF)")){ // #423
+			switch (tz390.fp_type){  // gen (min) hex for tz390.fp_type
+			case 0: // tz390.fp_db_type s1,e11,m52 with assumed 1
+				dc_hex = "7FF0000000000000";
+			    break;
+			case 1: // tz390.fp_dd_type s1,cf5,bxcf8,ccf50
+				dc_hex = "EEEEEEEEEEEEEEEE"; 
+				break;
+			case 2: // tz390.fp_dh_type s1,e7,m56 with hex exp
+				dc_hex = "EEEEEEEEEEEEEEEE";
+				break;
+			case 3: // tz390.fp_eb_type s1,e7,m24 with assumed 1
+	            dc_hex = "7F000000";
+	            break;
```

Comment:
According to tz390, case 3 is for BFP short. According to PoP 13, p19-2, the exponent is 8 bits (bits 1-8), so the value "7F000000" should be "7F800000".  Also, don't know what you mean by "with assumed 1" in the comment on line 11020.